cppsdk/source/playfab/PlayFabEvent.cpp

```diff
@@ -0,0 +1,11 @@
+#include <stdafx.h>
+
+#ifndef DISABLE_PLAYFABENTITY_API
+
+#include <playfab/PlayFabEvent.h>
+
+namespace PlayFab
```

Comment:
A single line comment around why this is empty ? :)

Reply:
It will be full in a moment.. :)

---
In reply to: [227948190](https://github.com/PlayFab/XPlatCppSdk/pull/6#discussion_r227948190) [](ancestors = 227948190)